### DISCUSSION TITLE
The error of not being able to interact with paper at the Inspector's position after hiden

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2460] [Q&A]
### Introduction

Why, after I have hidden this inspector, when I interact over this area, I can't interact with the paper, it seems like there is an issue here?

### Steps to reproduce

![123](https://github.com/clientIO/joint/assets/91791283/eaae335b-6ddf-4223-8b97-c2800e6acca2)
![456](https://github.com/clientIO/joint/assets/91791283/4a57ceb7-e717-4cc8-8cba-d952b70487cc)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
And how did you hide the inspector? Isn't there an invisible `<div/>` element that you are clicking on instead of interacting with the paper?

**Reply by QuangLanh** [reply]
I call this function to hide the inspector.
`joint.ui.Inspector.close();`
![image](https://github.com/clientIO/joint/assets/91791283/7e60d6e8-fea2-4baf-bcea-45a180b482ce)

**Reply by kumilingus** [reply]
It removes the inspector element, not the invisible container (the parent of original inspector).

**Reply by QuangLanh** [reply]
So what should I do to invisable it completely?

**Reply by kumilingus** [reply]
You can [remove](https://developer.mozilla.org/en-US/docs/Web/API/Element/remove) the `<div/>` or set [display=none](https://developer.mozilla.org/en-US/docs/Web/CSS/display).
